apps/react/src/epilog/form-adapters/contraceptivesAdapter.ts

```diff
@@ -131,18 +134,32 @@ export const formAdapter: FormAdapter<FormValues> = {
 
     return formValues;
   },
+
+  
   formValuesToEpilog: (values) => {
+    console.log("values",values);
```

Comment:
Probably it's a good idea to be more specific with what exactly we are logging here. Otherwise, it's hard for other developers (or even your future you) to derive meaning from this.